basler_params:
  -
    name: AcquisitionFrameRateAbs
    type: float
    value: 100.0
  -
    name: ExposureTimeAbs
    type: float
    value: 20000.0
  -
    name: AcquisitionMode
    type: enum
    value: Continuous
  -
    name: BalanceWhiteAuto
    type: enum
    value: 'Once'
  -
    name: TriggerMode
    type: enum
    value: 'Off'
  - 
    name: LineSelector
    type: enum
    value: 'Line2'
  -
    name: LineMode
    type: enum
    value: Output
  -
    name: LineSource
    type: enum
    value: 'ExposureActive'
  -
    name: GevSCPSPacketSize
    type: int
    value: 9000
  -
    name: GevSCPD
    type: int
    value: 9053
  -
    name: GevSCFTD
    type: int
    value: 9018
